src/presentation/http/router/user.test.ts

```diff
@@ -0,0 +1,46 @@
+import { describe, test, expect } from 'vitest';
+
+
+describe('user API', () => {
+  describe('GET /user/myself', () => {
+    test('returns user with status code 200 if user exists', async () => {
+      const expectedStatus = 200;
+      const userId = 1;
+      const accessToken = global.auth(userId);
+      const expectedUser =   {
+        'id': '1',
+        'email': 'a@a.com',
+        'name': 'Test user 1',
+        'photo': '',
+      };
+      const response = await global.api?.fakeRequest({
+        method: 'GET',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: '/user/myself',
+      });
+
+      expect(response?.statusCode).toBe(expectedStatus);
```

Comment:
let's from now on not use `expectedStatus`, `expectedUser` etc variables and place such data to expect expression directly